src/Controller/CollaborativeEditorsController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\collaborative_editors\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Url;
+
+/**
+ * Defines CollaborativeEditorsController class.
+ */
+class CollaborativeEditorsController extends ControllerBase {
+
+  /**
+   * Display the markup.
+   *
+   * @return array
+   *   Return markup array.
+   */
+  public function plugins() {
+    $table['plugins'] = [
+      '#type' => 'table',
+      '#header' => array(t('Plugin'), t('Operations')),
+      '#empty' => t('There are no plugins enabled.'),
```

Comment:
Follow coding standards.
`$this->t(`
and array should be represented using `[]`